server/lambda/layer/common/nodejs/custom_modules/logger.js

```diff
@@ -0,0 +1,15 @@
+const { createLogger, format, transports } = require('winston');
+
+const { combine, timestamp, json } = format;
+
+const logger = createLogger({
+  format: combine(
+    timestamp({
+      format: 'YYYY-MM-DD HH:mm:ss',
+    }),
+    json(),
+  ),
+  transports: [new transports.Console()],
+});
+
```

Comment:
in the interest of making this logger more useful. I think it makes sense to automatically log errors to Slack. Here is a [package](https://www.npmjs.com/package/winston-slack-webhook-transport) that allows us to create a transport.

Reply:
will be part of https://freshworks.atlassian.net/browse/ETS-521